### ISSUE TITLE
Elevated circulating aconitic acid concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#6031]
**Preferred term label:** Elevated circulating aconitic acid concentration

**Synonyms**

**Definition (free text, please give PubMed ID)** Concentration of aconitic acid in the blood is above the normal range., PMID: 31278494

**Parent term (use hpo.jax.org/app)** Abnormal circulating carboxylic acid concentration HP:0004354

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Disease annotation: https://omim.org/entry/614265

**Comment by pnrobinson** [comment]
adding as HP_0033179
